tests/components/energyzero/test_services.py

```diff
@@ -73,13 +82,18 @@ async def test_service(
 )
 async def test_service_validation(
     hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+    include_mock_entry: bool,
```

Comment:
Pass a dict with the config entry item instead, or empty dict, so that we can always update the service data.